app/assets/javascripts/module_loader.js

```diff
@@ -0,0 +1,36 @@
+(function() {
```

Comment:
We have already in `vendor/assets/javascripts` the file moj.js that serves this very same purpose. Why create a new one?